usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -66,8 +71,76 @@ def search_transactions(filters, fields, sort, order, lower_limit, limit):
     index_name = index_name[:-1]+'*'
     try:
         response = CLIENT.search(index=index_name, body=query)
-    except Exception as es1:
+    except Exception:
         return False, -1
     total = response['hits']['total']
     results = format_for_frontend(response['hits']['hits'])
-    return results, total
+    return results, total, transaction_type
+
+
+def get_total_results(keyword, index_name):
+    index_name = '{}-{}'.format(TRANSACTIONS_INDEX_ROOT,
+                                index_name.replace('_', ''))+'*'
+    query = {
+        'query': {
+            'query_string': {
+                'query': keyword
+                }
+            }
+    }
+    try:
+        response = CLIENT.search(index=index_name, body=query)
+        return response['hits']['total']
+    except Exception:
+        return -1
```

Comment:
It would be better to log the exception instead of simply returning -1. I would discuss with @ebdabbs to see if a -1 for some of the results would be acceptable. I gut tell me it is better to return a value of 0